point.py

```diff
@@ -0,0 +1,75 @@
+import math
+import random
+
+
+class Point:
+
+    def __init__(self, x = 0, y = 0, mark = []):
+        self.x = x
+        self.y = y
+        self.mark = mark
+
+    def check_coincident(self, b):
+
+        if b.x == self.x and b.y == self.y:
```

Comment:
With imports working, would it be possible to use code already written in the `utils` module?  Don't go back and alter anything, just think about if it is possible and why it might be preferable to use that approach?